src/assignment/views/AssignmentEdit.tsx

```diff
@@ -163,6 +159,43 @@ const AssignmentEdit: FunctionComponent<DefaultSecureRouteProps<{ id: string }>>
 		});
 	}, [assignment.blocks, setAssignment, setValue]);
 
+	const fragmentSwitchButtons = useCallback(
+		(block: AssignmentBlock) => [
+			{
+				active: !block.use_custom_fields,
+				label: t('assignment/views/assignment-edit___origineel'),
+				onClick: () => {
+					setBlock(block, {
+						use_custom_fields: false,
+					});
+				},
+			},
+			{
+				active: block.use_custom_fields && !isRichTextEmpty(block.custom_description),
+				label: t('assignment/views/assignment-edit___aangepast'),
+				onClick: () => {
+					setBlock(block, {
+						use_custom_fields: true,
+						custom_title: block.original_title || block.item?.title,
+						custom_description: block.original_description || block.item?.description,
+					});
+				},
+			},
+			{
+				active: block.use_custom_fields && isRichTextEmpty(block.custom_description),
```

Comment:
Wat gebeurt er als ik een custom titel wil, maar geen beschrijving?

Reply:
https://meemoo.atlassian.net/browse/AVO-1903?focusedCommentId=32831
```
Hierdoor ontstaat het gedrag dat wanneer een leerkracht “Aangepast“ selecteert en vervolgens het “Beschrijving fragment“ veld leeg maakt, de switch zal springen naar “Geen beschrijving“. Zie de onderstaande schermopname.
```

Reply:
ah, ok, got it